management/management.go

```diff
@@ -299,20 +299,20 @@ func (m *managementError) Status() int {
 	return m.StatusCode
 }
 
-// reqOption configures a call (typically to retrieve a resource) to Auth0 with
+// ReqOption configures a call (typically to retrieve a resource) to Auth0 with
 // query parameters.
-type reqOption func(url.Values)
+type ReqOption func(url.Values)
```

Comment:
Can we use the name `RequestOption` for this instead? As its becoming part of the public API maybe its okay to be more verbose.